### PR TITLE
Code Refactoring

### PR DESCRIPTION
**Pull request by edencakir**
We face some performance issues when binding functions at every render call, so I put some time on refactoring them, and also changed default exports into named exports.

### CODE REVIEW

File: src/screens/Profiles/ProfileScreen.js [hunk 1]
```diff
@@ -163,7 +173,7 @@ class ProfileScreen extends Component {
     return item;
   }
 
-  renderInfoBlock = () => {
+  renderInfoBlock() {
```
Comment (blargity) [inline src/screens/Profiles/ProfileScreen.js]: This style with the `= () => {` doesn't harm performance and I tend to use it throughout my components. In `render` methods it's not actually necessary, but in event handlers it is or your `this` is bound incorrectly so I tend to just use it everywhere. It still only creates the function once on instantiation of the component, so in general I'd leave this in place unless I'm missing something.

File: src/screens/Profiles/ProfileScreen.js [hunk 2]
```diff
@@ -234,7 +244,7 @@ class ProfileScreen extends Component {
     return result;
   }
 
-  renderHeader = () => {
+  renderHeader() {
```
Comment (blargity) [inline src/screens/Profiles/ProfileScreen.js]: Same here.

File: src/screens/Search/FilterSub.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ class FilterSub extends Component {
     this.renderItem = this.renderItem.bind(this);
     this.renderLength = this.renderLength.bind(this);
     this.renderFooter = this.renderFooter.bind(this);
+    this.renderSectionHeader = this.renderSectionHeader.bind(this);
```
Comment (blargity) [inline src/screens/Search/FilterSub.js]: All of these `.bind(this)` calls are exactly what that `whatever = () => {` style declarations are about, so you can actually get rid of all of these lines here and just swap the `onSubmit`, `renderSort`, `renderItem`, `renderLength`, `renderFooter`, and `renderSectionHeader` methods to be declared in that style and you can then get rid of the `constructor` entirely.

File: src/screens/Search/SearchFilter.js [hunk 4]
```diff
@@ -19,6 +19,7 @@ class SearchFilter extends Component {
     this.renderFooter = this.renderFooter.bind(this);
     this.renderCustomItem = this.renderCustomItem.bind(this);
     this.renderItem = this.renderItem.bind(this);
+    this.renderSectionHeader = this.renderSectionHeader.bind(this);
```
Comment (blargity) [inline src/screens/Search/SearchFilter.js]: Same here in terms of the `= () => {` style.

File: src/screens/Search/SearchScreen.js [hunk 5]
```diff
@@ -13,6 +12,8 @@ import { followUser } from 'kitsu/store/user/actions';
 import { captureUsersData } from 'kitsu/store/users/actions';
 import { ResultsList, TopsList } from './Lists';
 
+const TABBAR_HEIGHT = 40;
+
 const styles = {
```
Comment (blargity) [inline src/screens/Search/SearchScreen.js]: Would be great to make this `const styles = Stylesheet.create({`.

File: src/screens/Profiles/ProfileScreen.js (hunk 1 above)
Comment (edencakir) [reply src/screens/Profiles/ProfileScreen.js]: I agree but indeed most of these are not my work, so I was just formatting them to have a decent common code style. I'll change these to arrow versions 👍

**Comment by edencakir** [comment]
@blargity should be okay now

**Comment by blargity** [comment]
Thank you!

**Pull request merged**